### DISCUSSION TITLE
Fire Weather Forecast Product

### DISCUSSION THREAD

**Discussion by dsloanjr** [weather-gov/api#538] [Q&A]
Hi everyone,

I've been trying and failing to see if there is a way to get the information listed in the Routine Fire Wx Fcst.
I am able to get the information I am looking for through the raw data for the gridpoints. Is it possible to retrieve the information for the zone? Thank you!

https://forecast.weather.gov/product.php?issuedby=ILX&product=FWF&site=ilx

![image](https://user-images.githubusercontent.com/46930793/159966403-a87e6996-094b-402e-bdb5-bb9284199d78.png)

**Comment by sullynole** [comment]
> Hi everyone,
> 
> I've been trying and failing to see if there is a way to get the information listed in the Routine Fire Wx Fcst.
> I am able to get the information I am looking for through the raw data for the gridpoints. Is it possible to retrieve the information for the zone? Thank you!
> 
> https://forecast.weather.gov/product.php?issuedby=ILX&product=FWF&site=ilx
> 
> ![image](https://user-images.githubusercontent.com/46930793/159966403-a87e6996-094b-402e-bdb5-bb9284199d78.png)
> 
> 

A text formatter at the local office creates that text from the gridpoints within the ILZ027...etc. fire zone areas.  Each 2.5km square grid can only be accessed individually from the API gridpoints endpoints.  

If interested in the text forecast from the API, it can be found here:  https://api.weather.gov/products/types/FWF/locations/ILX

If interested in the geometry of the ILZ027...etc. fire weather zones, that be found like this:
https://api.weather.gov/zones/fire/ILZ027

**Reply by benghunt** [reply]
Is the question here

Why do the forecast products such as FWF or PFM contain so much more data than can be found at the gridpoint forecast endpoint

And why is it locked up in a weird text file that we have to write a custom parser for

Because that is my issue as well

**Reply by sullynole** [reply]
> Is the question here
> 
> Why do the forecast products such as FWF or PFM contain so much more data than can be found at the gridpoint forecast endpoint
> 
> And why is it locked up in a weird text file that we have to write a custom parser for
> 
> Because that is my issue as well 

The forecast endpoint has the forecast text, but the raw grid data with all fields are available in the forecastGridData endpoint, i.e., https://api.weather.gov/gridpoints/ILX/44,94
